api/end_device.proto

```diff
@@ -270,6 +270,8 @@ message MACSettings {
   // Maximum EIRP (dBm).
   // If unset, the default value from regional parameters specification will be used.
   DeviceEIRPValue desired_max_eirp = 30;
+  // The minimum duration passed before a network-initiated(e.g. Class B or C) downlink following an arbitrary downlink.
```

Comment:
I'd reframe this as 
```
The minimum duration that must pass before a network-initiated(e.g. Class B or C) downlink is sent following an arbitrary downlink.
```
This feels like a `downlink spacing` or a `scheduling delay` rather than an `interval` but that could just be me.

Btw, what is an arbitrary downlink?